ouroboros-network-protocols/src/Ouroboros/Network/Protocol/ChainSync/PipeClient.hs

```diff
@@ -0,0 +1,70 @@
+{-# LANGUAGE DataKinds           #-}
+{-# LANGUAGE GADTs               #-}
+{-# LANGUAGE NamedFieldPuns      #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE RankNTypes #-}
+
+module Ouroboros.Network.Protocol.ChainSync.PipeClient
+where
+
+import Network.TypedProtocol
+import Ouroboros.Network.Protocol.ChainSync.Client
+import Ouroboros.Network.Protocol.ChainSync.Type
+import Pipes
+
+data ChainSyncResponse header point tip
```

Comment:
Suggest renaming `header` everywhere to `block`. This will be intended to be used with blocks. With the node-to-client protocol, the chain sync provides blocks (whereas for the node-to-node it's used at type header).